src/main/java/com/smartystreets/api/RetrySender.java

```diff
@@ -19,9 +18,19 @@ public RetrySender(int maxRetries, Sleeper sleeper, Logger logger, Sender inner)
         this.inner = inner;
     }
 
-    public Response send(Request request) throws SmartyException, IOException {
+    public Response send(Request request) throws SmartyException, IOException, InterruptedException {
         for (int i = 0; i <= this.maxRetries; i++) {
             Response response = this.trySend(request, i);
+            if (response instanceof TooManyRequestsResponse) {
+                long wait = ((TooManyRequestsResponse) response).getHeaders().firstValueAsLong("Retry-After").orElse(10L);
+                if (wait < 1) {
+                    wait = 1L;
+                }
+                this.logger.log("The rate limit for requests has been exceeded. Sleeping " + wait + " seconds...");
```

Comment:
If this could occur frequently, there needs to be a way to disable this log message via a logging level.

Reply:
I'll just comment this line out for now.